tests/grpc/call_test.go

```diff
@@ -110,14 +111,11 @@ func TestCall(t *testing.T) {
 		assert.Equal(t, true, req.AcceptsOutputValues)
 
 		return p.CallResponse{
-			Return: resource.PropertyMap{
-				"r1": resource.NewProperty(resource.Output{
-					Element: resource.NewProperty("e1"),
-					Dependencies: []resource.URN{
-						"urn1", "urn2",
-					},
+			Return: property.NewMap(map[string]property.Value{
+				"r1": property.New(property.Computed).WithDependencies([]resource.URN{
```

Comment:
One issue I have with `resource.PropertyValue` is that it makes it very easy to construct the wrong thing. Here, the `Element: resource.NewProperty("e1")` doesn't do anything since `Known: true` was not set. This value is correctly translated to `property.New(property.Computed)`.

Reply:
🤦‍♂️ something something structs and default zeros